src/solc_0.8/raffle/GenericRaffle.sol

```diff
@@ -144,6 +165,9 @@ contract GenericRaffle is ERC721EnumerableUpgradeable, OwnableUpgradeable, Reent
         }
     }
 
+    /* solhint-disable-next-line no-empty-blocks */
+    function renounceOwnership() public virtual override onlyOwner {}
```

Comment:
probably should revert to avoid tx

Reply:
@mvanmeerbeck Not sure what you mean by this